configs/sandpack-contents/tutorial/package.ts

```diff
@@ -10,6 +10,7 @@ export const packageJson = `
   "main": "src/index.js",
   "dependencies": {
     "@chakra-ui/react": "1.8.6",
+    "@chakra-ui/icons": "1.1.7",
```

Comment:
I assume this dependency is one you want to use in your tutorial? You need to add it here for it to work properly in the sandbox: https://github.com/chakra-ui/chakra-ui-docs/blob/main/src/components/tutorial/tutorial-container.tsx#L85-L91

Reply:
This package.json is just an example file and has no real impact on the sandbox.